_drafts/2015-10-02-microcoap-and-ff-copper.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: CoAP on RIOT
+subtitle: implementing and testing with microcoap and copper
+categories: CoAP
+author: lotte
+date: 2015-03-17
+---
+
+In [one of our last posts](http://watr.li/what-is-coap.html), we explained how the [Constrained Application Protocol](http://coap.technology) enables us to exchange data between nodes in the Internet of Plants using a request/reply cycle similar to that of HTTP. 
```

Comment:
* I'd write "In a previous post". 
* how the Constrainted application... (CoAP) (wuerde ich in klammern hinzufuegen)